crates/sui-indexer-alt/src/handlers/coin_balance_buckets_pruner.rs

```diff
@@ -66,3 +101,7 @@ impl Handler for CoinBalanceBucketsPruner {
         Ok(rows_deleted)
     }
 }
+
+impl FieldCount for CoinBalanceBucketToBePruned {
```

Comment:
This needs a bit more explanation -- naively, this should be `2` because that's how many fields would have been bound, but we are not binding anything in the use of this type. Could we add a comment here explaining that?